Effects/FlxWaveSprite/source/PlayState.hx

```diff
@@ -88,6 +107,18 @@ class PlayState extends FlxState
 		_waveSprite.center = Std.int(FlxMath.bound(_waveSprite.center, 0, _waveSprite.height));
 		_waveSprite.strength = Std.int(FlxMath.bound(_waveSprite.strength, 0, 500));
 		_waveSprite.speed = FlxMath.bound(_waveSprite.speed, 0, 80);
+		_waveSprite.wavelength = Std.int(FlxMath.bound(_waveSprite.wavelength, 1, 20));
+	}
+	
+	private function incrementDirection():Void
+	{
+		switch (_waveSprite.direction)
```

Comment:
This could be written a bit more elegantly by using `switch` as an expression. See [last section on this page](http://haxeflixel.com/documentation/code-style/), except you do an assignment instead of `return`:

`_waveSprite.direction = switch (`